app/main.py

```diff
@@ -1 +1,58 @@
-#  write your code here
+class Animal:
+    animals = []
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.is_hungry = is_hungry
+        self.appetite = appetite
+        self.animals.append(self)
```

Comment:
`animals` is a class attribute, call it from the `Animal` class, not from `self`.